types/feathersjs__express/index.d.ts

```diff
@@ -10,12 +11,18 @@ import * as express from 'express';
 export default function feathersExpress<T>(app: FeathersApplication<T>): Application<T>;
 export type Application<T> = express.Application & FeathersApplication<T>;
 
-export function errorHandler(options?: any): express.ErrorRequestHandler;
+export function errorHandler(options?: {
+    public?: string,
+    logger?: { error?: (msg: string) => void },
+    html?: any,
+    json?: any,
+}): express.ErrorRequestHandler;
 export function notFound(): express.RequestHandler;
-export const rest: {
-    (): () => void;
-    formatter: express.RequestHandler;
-};
+
+export function rest(): (handler?: express.RequestHandler) => void;
+export namespace rest {
```

Comment:
I'd rather keep the callable-object signature, because the function/namespace construct is actually more of a hack for commonJS exports which will soon become obsolete due to TS 2.7's `esModuleInterop`